src/mach/templates/partials/aws.tf

```diff
@@ -34,6 +34,10 @@ provider "aws" {
   {% include 'partials/endpoints/aws_url_locals.tf' %}
 {% endif %}
 
+{% if aws.cloud_front %}
```

Comment:
Shall we name this `cloudfront`? More in line with how it's named in Terraform provider

Reply:
agreed!